addons/crm/wizard/crm_lead_to_opportunity.py

```diff
@@ -32,12 +32,15 @@ def default_get(self, fields):
     action = fields.Selection([
         ('create', 'Create a new customer'),
         ('exist', 'Link to an existing customer'),
-        ('nothing', 'Do not link to a customer')
-    ], string='Related Customer', compute='_compute_action', readonly=False, store=True, compute_sudo=False)
+    ], string='Related Customer', compute='_compute_action',
+    precompute=True, readonly=False, required=True, store=True, compute_sudo=False)
     lead_id = fields.Many2one('crm.lead', 'Associated Lead', required=True)
     duplicated_lead_ids = fields.Many2many(
         'crm.lead', string='Opportunities', context={'active_test': False},
         compute='_compute_duplicated_lead_ids', readonly=False, store=True, compute_sudo=False)
+    company_partner_id = fields.Many2one(
```

Comment:
Confusing to have the same field name after reading previous commit :dizzy_face: